### ISSUE TITLE
Enable Location Updates issue

### ISSUE THREAD

**Issue by diver7** [MicrosoftDocs/powerapps-docs#1145]
Adding the code for location updates to the submit button (btnDateRange) causes the app to hang when the button is pressed.

**Comment by mr-dang-msft** [comment]
Hi @diver7,
Can you let us know if you're using the latest version of the app? 

As of this message, that is version V2.20200312.1708 (deployed 5pm PST on 3/12):
https://github.com/microsoft/powerapps-tools/commit/a084ebe04d15f506edb032bcc8bda0b7dbead52a

By 'hangs' do you see dots going across the top of the screen?

**Comment by diver7** [comment]
No sure where to find that version number you stated but I downloaded the package for the app from GitHub on 3/13/2020 at 2:51PM CDT. So I should have the latest. 

When you tap the Submit button it turns grey to acknowledge it has been hit but then nothing happens. Nothing on the screen changes and there are no progress indicators.

**Comment by diver7** [comment]
I found the issue. I was copying the location code from[ here](https://docs.microsoft.com/en-us/powerapps/maker/canvas-apps/sample-crisis-communication-app#optional-enable-location-updates
). I went back and just uncommented the two lines of code in the original app code for the submit button and it worked fine. 

```
            // To implement location, add a comma to the line above and uncomment the lines below for latitude and longitude.
             Latitude: Text(Location.Latitude),
             Longitude: Text(Location.Longitude)
```

Looks like the article on docs.microsoft.com just needs to be updated with the latest code.
